lib/pairmotron.ex

```diff
@@ -4,6 +4,7 @@ defmodule Pairmotron do
 
   # See http://elixir-lang.org/docs/stable/elixir/Application.html
   # for more information on OTP Applications
+  @spec start(atom(), map()) :: Supervisor.on_start
```

Comment:
You could definitely make an argument that these didn't need Typespecs, but they don't hurt, and it was fun tracking them down.